src/services/postService.js

```diff
@@ -1,12 +1,41 @@
 const { throwIfError } = require('../utilities/dynamoUtilities');
-const { sendPost } = require("../repository/postDAO");
+const postDAO = require("../repository/postDAO");
+const uuid = require("uuid");
 
 async function createPost(username, text, score, title){
-    const post = await sendPost(username, text, score, title);
-    throwIfError(post);
-    return {message: "Post created successfully"};
+    const post = {class: "post", itemID: uuid.v4(), by: username, desc: text, score, title, replies: []};
+    const data = await postDAO.sendPost(post);
+    throwIfError(data);
+    return post;
+}
+
+async function seePosts(){
+    //TODO try to see if better option than scan
+    return await postDAO.scanPosts();
+}
+
+async function createReply(username, text, id){
+    //TODO check if post ID is valid
```

Comment:
I'd implement this before merging. Here you wouldn't need scan, you can just use query with the primary key of  itemID and class I believe and check whether you get an item back or not